ingestion/functions/parsing/brazil_distrito_federal/distrito_federal.py

```diff
@@ -0,0 +1,196 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            "common/python"))
+    import parsing_lib
+
+_AGE = "Faixa Etária"
+_GENDER = "Sexo"
+_MUNICIPALITY = "RA"
+# UF stands for Unidade Federada which is the same as State
+_STATE = "UF"
+_DATE_CONFIRMED = "Data Cadastro"
+_DATE_SYMPTOMS = "dataPrimeirosintomas"
+_DEATH = "Óbito"
+_LUNG = "Pneumopatia"
+_KIDNEY = "Nefropatia"
+_HEMATOLOGIC = "Doença Hematológica"
+_METABOLIC = "Distúrbios Metabólicos"
+_IMMUNOSUPPRESSED = "Imunopressão"
+_OBESITY = "Obesidade"
+_OTHERS = "Outros"
+_CARDIOVASCULAR = "Cardiovasculopatia"
+
+_COMORBIDITIES_MAP = {
+    "Pneumopatia": "lung disease",
+    "Nefropatia": "kidney disease",
+    "Distúrbios Metabólicos": "disease of metabolism",
+    "Obesidade": "obesity",
+    "Cardiovasculopatia": "cardiovascular system disease"
+}
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+    """
+    # Some date fields are empty
+    if raw_date:
+        # There is variation in how dates are reported
+        if "-" in raw_date:
+            date = datetime.strptime(raw_date, "%Y-%m-%d")
+        else:
+            date = datetime.strptime(raw_date, "%d/%m/%Y")
+        return date.strftime("%m/%d/%YZ")
+    else:
+        return None
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "Masculino":
+        return "Male"
+    if raw_gender == "Feminino":
+        return "Female"
+
+
+def convert_events(date_confirmed, date_symptoms, death):
+    """There is no date of death"""
+    events = [
+        {
+            "name": "confirmed",
+            "dateRange": {
+                "start": convert_date(date_confirmed),
+                "end": convert_date(date_confirmed),
+            },
+        }
+    ]
+    if date_symptoms:
+        events.append(
+            {
+                "name": "onsetSymptoms",
+                "dateRange": {
+                    "start": convert_date(date_symptoms),
+                    "end": convert_date(date_symptoms),
+                },
+            }
+        )
+    if death == "Sim":
+        events.append(
+            {
+                "name": "outcome",
+                "value": "Death",
+            }
+        )
+    return events
+
+
+def convert_preexisting_conditions(lung: str, kidney: str, metabolic: str,
+                                   cardiovascular: str, obesity: str):
+    preexistingConditions = {}
+    if all((lung, kidney, metabolic, cardiovascular, obesity)) == "Sim":
+        return None
+
+    preexistingConditions["hasPreexistingConditions"] = True
+    commorbidities_list = []
```

Comment:
nit, just commorbidities, avoid setting the type of your vars in their names it doesn't really help readability